### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by apexskier** [apexskier/dolphin-controller#3] [Announcements]
First release

<hr /><em>This discussion was created from the release <a href='https://github.com/apexskier/dolphin-controller/releases/tag/'>v1.0.0</a>.</em>

**Comment by thatcherclough** [comment]
Works great! Very well put together!

**Comment by j0shuaisrael** [comment]
I haven't set it up yet, but i like the clever design. thanks for developing and making this available.
